src/main/java/fi/aalto/cs/apluscourses/utils/BuildInfo.java

```diff
@@ -20,6 +20,9 @@
   @NotNull
   public final Version version;
 
+  @NotNull
+  public final Version courseVersion;
```

Comment:
Once again not hugely important, but renaming the other one to `pluginVersion` would help disambiguate the two